data/pc/1.6.4/protocol.json

```diff
@@ -0,0 +1,2290 @@
+              {
+                "compareTo": "mode",
+                "fields": {
+                  "0": "string",
+                  "2": "string"
+                },
+                "default": "void"
+              }
+            ]
+          },
+          {
+            "name": "friendlyFire",
+            "type": [
+              "switch",
+              {
+                "compareTo": "mode",
+                "fields": {
+                  "0": "i8",
+                  "2": "i8"
+                },
+                "default": "void"
+              }
+            ]
+          },
+          {
+            "name": "players",
+            "type": [
+              "switch",
+              {
+                "compareTo": "mode",
+                "fields": {
+                  "0": [
+                    "array",
+                    {
+                      "countType": "i16",
+                      "type": "string"
+                    }
+                  ],
+                  "3": [
+                    "array",
+                    {
+                      "countType": "i16",
+                      "type": "string"
+                    }
+                  ],
+                  "4": [
+                    "array",
+                    {
+                      "countType": "i16",
+                      "type": "string"
+                    }
+                  ]
+                },
+                "default": "void"
+              }
+            ]
+          }
+        ]
+      ],
+      "packet_custom_payload": [
+        "container",
+        [
+          {
+            "name": "channel",
+            "type": "string"
+          },
+          {
+            "name": "data",
+            "type": [
+              "buffer",
+              {
+                "countType": "i16"
+              }
+            ]
+          }
+        ]
+      ],
+      "packet_encryption_key_response": [
+        "container",
+        [
+          {
+            "name": "sharedSecret",
+            "type": [
+              "buffer",
+              {
+                "countType": "i16"
+              }
+            ]
+          },
+          {
+            "name": "verifyToken",
+            "type": [
+              "buffer",
+              {
+                "countType": "i16"
+              }
+            ]
+          }
+        ]
+      ],
+      "packet_encryption_key_request": [
+        "container",
+        [
+          {
+            "name": "serverId",
+            "type": "string"
+          },
+          {
+            "name": "publicKey",
+            "type": [
+              "buffer",
+              {
+                "countType": "i16"
+              }
+            ]
+          },
+          {
+            "name": "verifyToken",
+            "type": [
+              "buffer",
+              {
+                "countType": "i16"
+              }
+            ]
+          }
+        ]
+      ],
+      "packet_server_list_ping": [
```

Comment:
Still have the issue that this packet is not implemented at all here.

Reply:
How should I proceed?

Reply:
I documented it here, https://github.com/PrismarineJS/minecraft-data/pull/840#discussion_r1473553320 although the best way would be to open up 1.6.4 source and do it following that.